lib/web/apiserver.go

```diff
@@ -190,7 +192,7 @@ func (h *RewritingHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.appHandler.ServeHTTP(w, r)
 		return
 	}
-	if redir, ok := app.HasName(r, h.publicAddr); ok {
```

Comment:
nit: after switching to `cfg.ProxyPublicAddrs` the `publicAddr` filed seems to be used and can be removed.